scripts/bundleSize/bundleSize.test.js

```diff
@@ -1,30 +1,214 @@
-import ora from 'ora';
-import chalk from 'chalk';
-import { readdirSync, statSync } from 'fs';
+/* eslint-disable import/extensions */
 import stripAnsi from 'strip-ansi';
-
-jest.mock('./pageTypeBundleExtractor');
-jest.mock('ora');
-jest.mock('chalk', () => ({
-  red: a => a,
-  green: a => a,
-  blue: a => a,
-  yellow: a => a,
-  cyan: { bold: b => b },
-  bold: a => a,
-}));
-jest.mock('../../cypress/support/config/services', () => ({
-  service1: {},
-  service2: {},
+import { jest } from '@jest/globals';
+
+jest.unstable_mockModule('./pageTypeBundleExtractor', () => ({
+  extractBundlesForPageType: pageType => {
```

Comment:
Are we able to revert to using the contents of the mock instead of hardcoding here?

Reply:
As we worked on this we couldn't find a way of using `__mocks__` folders, you can infer they aren't supported yet as the docs for ESM in jest describe that a factory function is mandatory: https://jestjs.io/docs/ecmascript-modules 

This means the previous automated behaviour of `jest.mock('./pageTypeBundleExtractor')` won't work as the factory function replaces any automated behaviour such as auto-mocking and using manual mocks in `__mocks__` folders.

This issue tracks the development of this API: https://github.com/jestjs/jest/issues/9430

Should we extract the mock module to a seperate file to make this file more readable and deleted the `__mocks__` folder for now?

Reply:
> Should we extract the mock module to a seperate file to make this file more readable and deleted the `__mocks__` folder for now?

Yes, this probably sounds like the best approach. Can we leave it in the `__mocks__` folder and import it? Or perhaps rename `__mocks__` to `mocks`?

Reply:
I've kept the `__mocks__` folder for the moment, hope it looks okay?